### DISCUSSION TITLE
Baklava v2 release

### DISCUSSION THREAD

**Discussion by newcat** [newcat/baklavajs#333] [Announcements]
After pretty much exactly a year in beta, Baklava v2 is now officially released 🎉 Since most of you are already using the beta anyway, not much will change.

### What does this release mean?
- There won't be any breaking changes in v2
- When installing BaklavaJS, the default is now v2, so no `@next` tag is needed anymore
- Features and bugfixes will be published in future minor/patch versions (2.x.x)
- BaklavaJS v1 enters maintenance mode - there won't be any new versions unless there are major bugs or security flaws
- The same also applies to the Vue 3 version of Baklava v1

As most of you have probably noticed, the development of BaklavaJS has stalled in the last couple of months. This will (hopefully) improve a little bit now; however, I still won't have much time to work on BaklavaJS.

If you want to see a rough roadmap, have a look here: https://github.com/newcat/baklavajs/projects/2

Feel free to post any questions or feedback in the comments of this discussion.

**Comment by lpequignot** [comment]
Hi, 

I've tested it this morning, works well ! 
One thing I noticed is that now we see the NodeInterface component for output (even if connected)
I thinks the change happen with the following commit : https://github.com/newcat/baklavajs/commit/485a01eee4e60b5ab19ae3df8170bbe25761c7b7

Is there a possibility of never show component for outputs ?

Thank you,
Louise

**Reply by newcat** [reply]
You can call `.setComponent(undefined)` on the `NodeInterface` as a workaround for now. I might add a global option in the future to configure the display behavior of node interface components.

**Reply by lpequignot** [reply]
The workaround works well, 
thank you.
